### PR TITLE
Update authors.yml & adding Dev Plan Changes post

### PR DESCRIPTION
**Pull request by havi**
adding author Ryan Stewart

This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [ ] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_data/authors.yml [hunk 1]
```diff
@@ -1,3 +1,11 @@
+full_name: Ryan Stewart
```
Comment (bdemers) [inline _source/_data/authors.yml]:
```suggestion
ryan-stewart:
  full_name: Ryan Stewart
```

File: _source/_data/authors.yml [hunk 2]
```diff
@@ -1,3 +1,12 @@
+ryan-stewart:
+  full_name: Ryan Stewart
+  display_name: Ryan Stewart
+  avatar: avatar-ryan-stewart.jpeg
+  web: http://ryanstewart.net
+  github: https://github.com/ryanstewart
+  bio:
+  Director of Product Management for CIAM Developer Platform at @okta
```
Comment (bdemers) [inline _source/_data/authors.yml]:
```suggestion
  bio: "Director of Product Management for CIAM Developer Platform at @okta"
```

**Review by mraible — APPROVED** [review]

File: _source/_posts/2021-10-05-okta-dev-plan-changes.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+layout: blog_post
+title: Okta Developer Plan Changes
+author: ryan-stewart 
+by: internal-contributor
+communities: [.net,java,javascript,go,php,python,ruby,devops,announcement]
+description: "This announces how we are streamlining and clarifying our developer product offering in light of the Auth0 acquisition. We are sunsetting Okta's Developer Advanced plan, and providing Auth0 plans for developers seeking a fully self-service experience."
```
Comment (mraible) [inline _source/_posts/2021-10-05-okta-dev-plan-changes.md]: This description might be a bit lengthy. It's recommended descriptions are under 150 characters. https://moz.com/learn/seo/meta-description

Comment (havi) [reply _source/_posts/2021-10-05-okta-dev-plan-changes.md]:
```suggestion
description: "In light of the Auth0 acquisition, we are streamlining Okta's developer product offering by retiring Okta's Developer Advanced plan, and providing Auth0 plans for developers seeking a fully self-service experience."
```

**Pull request merged**